Simplenote/Classes/SPTagListViewCell.m

```diff
@@ -1,154 +1,82 @@
 #import "SPTagListViewCell.h"
-#import "VSThemeManager.h"
-#import "SPBorderedView.h"
 #import "Simplenote-Swift.h"
 
 
+@interface SPTagListViewCell ()
+@property (nonatomic, strong) IBOutlet UIImageView *leftImageView;
```

Comment:
Still thinking about rtl languages and UI flipping.  Would it be just as descriptive to say something like `iconImageView` and avoid the left/right specifier?

Reply:
Much better, I'll also patch the textField's name! Thank you!!